pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -581,6 +581,14 @@ type podActions struct {
 	UpdatePodResources bool
 }
 
+// podLevelResources holds the set of resources applicable to the running pod
+type podLevelResources struct {
```

Comment:
This doesn't need to be a separate type from container resources. Just reuse `containerResources` for now, and leave a TODO to rename it to something more generic.